plugin/src/test/java/com/epam/grandhackathon/deployment/sphere/plugin/DeployMetaDataParameterValueTest.java

```diff
@@ -0,0 +1,54 @@
+package com.epam.grandhackathon.deployment.sphere.plugin;
+
+import static org.junit.Assert.assertNotNull;
+import hudson.EnvVars;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.epam.grandhackathon.deployment.sphere.plugin.listener.DeployVersionMetaDataListenerTest;
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.Constants;
+
+public class DeployMetaDataParameterValueTest {
+	private static final Logger LOGGER = LoggerFactory.getLogger(DeployVersionMetaDataListenerTest.class);
+    
+	@Rule
+    public JenkinsRule j = new JenkinsRule();
+	private String environmentKey = "qa";
+	private String buildVersion = "0.1";
+	private String applicationName = "app1";
+	
+	@Test
+	public void testBuildEnvVars() throws Exception {
```

Comment:
usually test starts with "should" e.g. shouldPopulateEnvironmentVariables for example